alternator/executor.cc

```diff
@@ -221,6 +221,18 @@ static std::optional<std::string> find_table_name(const rjson::value& request) {
     return table_name;
 }
 
+static bool should_add_capacity(const rjson::value& request) {
+    const rjson::value* return_consumed = rjson::find(request, "ReturnConsumedCapacity");
+    if (!return_consumed) {
+        return false;
+    }
+    if (!return_consumed->IsString()) {
+        throw api_error::validation("Non-string ReturnConsumedCapacity field in request");
+    }
+    std::string consumed = return_consumed->GetString();
+    return consumed == "TOTAL";
```

Comment:
What about 'INDEXES' ?

Reply:
Right. Please check explicitly for the supported strings - TOTAL, and NONE, for INDEXES throw an error saying this isn't supported yet, and for anything else, throw an error. Please add a test that all of this error path actually works (and you can run the same test against DyanamoDB to see that for ReturnedConsumedCapacity='dog' you get the same error.

Alternatively, you *can* support INDEXES for the GetItems you support - easily. Just please write a test for the INDEXES case, to make sure you know what DynamoDB really returns in that case.